.github/workflows/test.yaml

```diff
@@ -0,0 +1,32 @@
+name: test
+on: pull_request
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          override: true
+          toolchain: stable
+          profile: minimal
+      - uses: Swatinem/rust-cache@v2
+      - name: Install jq
+        uses: dcarbone/install-jq-action@v1.0.1
+      - name: 'Check jq'
+        run: |
+          which jq
+          jq --version
+          export JQ_LIB_DIR=$(eval which jq)
+      - name: Build workspace
+        run: export JQ_LIB_DIR=$(eval which jq) && cargo build
+      # - name: Test workspace
+      #   run: export JQ_LIB_DIR=$(eval which jq) && cargo test
```

Comment:
If the tests ran, this would likely fix #31

Reply:
Oh, and if this is commented out because the tests currently fail, that's fine. I think I may have merged a PR that could have broken the tests (this is why I want them running on PRs!)

If this is the case, I guess just let me know and leave it commented out. I'll fix it up separately.

Reply:
yes I saw the issue #31 which is how i decided to work on this.
But yes, the tests fail you can see the logs here https://github.com/suryapandian/jq-rs/actions/runs/6136985715
Once you fix the tests we could enable it.